### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### LINKED ISSUE #51: add a PR template
A simple and brief PR template will help the contributors and maintainers while developing **LeetEasy**. I would like to add a simple PR template as below

## example template

```markdown
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] branch name follows the branch naming convention mentioned [here](https://github.com/sudiptob2/leet-easy/blob/main/docs/BRANCH-NAMING.md)
- [ ] commit messages to follow the format mentioned in [contributing.md](https://github.com/sudiptob2/leet-easy/blob/main/docs/CONTRIBUTING.md)
- [ ] I have tested my code using `bash .github/check.sh`
- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
```

### PR DESCRIPTION
**Pull request by hari-ajidesh**
Closes #51

### CODE REVIEW

**Comment by hari-ajidesh** [comment]
Please add the hacktoberfest-accepted label to this issue

**Comment by sudiptob2** [comment]
Thanks for adding a lovely PR template. However, I actually want the template mentioned in the description of issue #52. Can you update :pray:

**Comment by hari-ajidesh** [comment]
PR template has been updated as requested in #51 . Do look into it

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,52 +1,24 @@
...
+Please delete options that are not relevant.
 
-<!-- Please do not submit updates to dependencies unless it fixes an issue. --> 
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
 
-<!-- Please try to limit your pull request to one type, submit multiple pull requests if needed. --> 
+# Checklist:
```
Comment (sudiptob2) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Checklist:
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,52 +1,24 @@
-<!-- Please refer to our contributing documentation for any questions on submitting a pull request, or let us know here if you need any help: https://github.com/sudiptob2/leet-easy/blob/main/docs/CONTRIBUTING.md -->
+# Description
```
Comment (sudiptob2) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Description
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,52 +1,24 @@
...
-## Pull request checklist
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
-Please check if your PR fulfills the following requirements:
-- [ ] Tests for the changes have been added (for bug fixes / features)
-- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
-  - Some docs updates need to be made in the `ionic-docs` repo, in a separate PR. See the [contributing guide](https://github.com/sudiptob2/leet-easy/blob/main/docs/CONTRIBUTING.md) for details.
+Fixes # (issue)
 
+## Type of change
```
Comment (sudiptob2) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Type of change
```

**Review by sudiptob2 — CHANGES_REQUESTED** [review]

**Comment by sudiptob2** [comment]
Can you please incorporate the change that I requested? I think **heading 3** looks good than **Heading 1**

**Review by sudiptob2 — COMMENTED** [review]
Can you please incorporate the change that I requested? I think heading 3 looks good than Heading 1

**Comment by hari-ajidesh** [comment]
Sorry for my mistake. I thought I incorporated them by clicking on "resolve conversation". Now they are updated

**Review by sudiptob2 — APPROVED** [review]

**Pull request merged**